Mavlink/MAVLinkInterface.cs

```diff
@@ -464,61 +463,31 @@ private void OpenBg(object PRsender, bool getparams, ProgressWorkerEventArgs pro
 
                     Thread.Sleep(1);
 
-                    // can see 2 heartbeat packets at any time, and will connect - was one after the other
+                    // listen for heartbeats from component id 1 (flight controller) only ensures that at least one flight controller is operational

```

Comment:
if you are connecting straight to a gimbal, without a flight controler whats the expected result?

Reply:
The expected result would be that it times out. What would be the use case of connecting a gimbal without a flight controller?

Reply:
@JWangAeronavics, in view of other changes which you mentioned @meee1 had made to handling components other than the flight controller, how much of this is still relevant?